### PR TITLE
Adding the zap talk video and fuzzer section.

### PR DESCRIPTION
**Pull request by DavidEBest**
As described on the [trello card](https://trello.com/c/RhXjzX9N/27-do-an-engineering-talk-demo-on-zap), this is an effort to get more out of the tech talk I did on OWASP ZAP.

### CODE REVIEW

File: _pages/security/dynamic-scanning.md [hunk 1]
```diff
@@ -53,10 +53,18 @@ Authentication can be handled in several ways; the following links provide guida
...
+
+["Fuzzing"](https://en.wikipedia.org/wiki/Fuzz_testing) refers to feeding a large amount of random (and/or potentially malicious) data to an application with the intention of finding vulnerabilities related to poor error handling or incomplete input validation. 
+
+Any request in ZAP can be fuzzed. Simply right click on it, select Attack -> Fuzzer. Read more about ZAP's Fuzzing capabilities [here](https://github.com/zaproxy/zap-core-help/wiki/HelpAddonsFuzzConcepts).
+
 ### More Information
 
 The [ZAP User Guide](https://github.com/zaproxy/zap-core-help/wiki) is phenomenal. If you run into an issue, this should be the first place you check.
 
+We gave an introduction to ZAP talk as part of our engineering tech talks series. [Vulnerability with OWASP ZAP](https://www.youtube.com/watch?v=2Dp7pAvKHaM) ([more info and slides](https://github.com/18F/tech-talks/tree/master/vuln-scanning))
```
Comment (afeld) [inline _pages/security/dynamic-scanning.md]: I'd feature this more prominently, up near the top!

https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#videos

File: _pages/security/dynamic-scanning.md [hunk 2]
```diff
@@ -53,10 +53,18 @@ Authentication can be handled in several ways; the following links provide guida
 
 The Attack Script runs the Active Scan and Spider functions. If your page is a javascript-driven single page application, you will want to run the Ajax Spider in addition to the standard spider. The Ajax Spider will instantiate a browser window and spider the application visually. This is significantly slower than the standard spider. Read more about the Ajax Spider [here](https://github.com/zaproxy/zap-core-help/wiki/HelpAddonsSpiderAjaxConcepts).
 
+#### Fuzzing
+
+["Fuzzing"](https://en.wikipedia.org/wiki/Fuzz_testing) refers to feeding a large amount of random (and/or potentially malicious) data to an application with the intention of finding vulnerabilities related to poor error handling or incomplete input validation.
```
Comment (afeld) [inline _pages/security/dynamic-scanning.md]: I'd add something to the effect of "basically, use this anywhere you rely on query param or have a form".

**Comment by DavidEBest** [comment]
@afeld Ready to be reviewed again. I incorporated your suggestions.

File: _pages/security/dynamic-scanning.md [hunk 3]
```diff
@@ -11,6 +11,10 @@ parent: Security
 
 You will need a running application to test, which you will want to be as production-like as possible. The 'Attack' scripts shouldn't do anything destructive, but will cause a spike in requests and errors. **Inform your team and #devops before running it on live (not locally-hosted) sites.**
 
+We gave an introduction to ZAP talk as part of our engineering tech talks series. [Vulnerability with OWASP ZAP](https://www.youtube.com/watch?v=2Dp7pAvKHaM) ([more info and slides](https://github.com/18F/tech-talks/tree/master/vuln-scanning))
```
Comment (afeld) [inline _pages/security/dynamic-scanning.md]: Not a blocker, but I would use the preview image syntax to give this a bigger call-out.

https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#videos

**Comment by afeld** [comment]
Just the one minor thing—feel free to merge afterwards!

**Pull request merged**